src/meta/app/src/share/share.rs

```diff
@@ -611,3 +628,38 @@ pub struct ShareInfo {
     pub name_ident: ShareNameIdent,
     pub meta: ShareMeta,
 }
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, Default, Eq, PartialEq)]
+pub struct ShareTableSpec {
+    pub name: String,
+    pub id: u64,
+    pub location: String,
+    pub version: u64,
```

Comment:
the version field can be moved to the top of share config file?

IMO we can keep the config evolution at the top level.